### PR TITLE
Random colors

### PR DESCRIPTION
**Pull request by sohilpandya**
Random colors are now added for each individual project. 
screen shot 2017-04-18 at 14 39 25

Related to #18 

Please review and merge after #25 & #28

### CODE REVIEW

File: time/AddTaskViewController.swift [hunk 1]
```diff
@@ -43,12 +34,11 @@ class AddTaskViewController: UIViewController, UITextFieldDelegate {
         // Dispose of any resources that can be recreated.
     }
     @IBAction func saveButton(_ sender: UIBarButtonItem) {
-        print("KKKKKK")
         let name = taskName.text ?? ""
                 let time = 12345667
-                let red = 123.0
-                let green = 223.0
-                let blue = 34.0
+                let red = drand48()
```
Comment (nelsonic) [inline time/AddTaskViewController.swift:39]: I would prefer to use a fixed set of _attractive_ colors e.g: http://flatuicolors.com than to "_randomise_" them.

**Review by nelsonic — APPROVED** [review]
@sohilpandya this looks good.
In a _future_ iteration I would recommend picking 20 _nice_ colors from https://www.materialui.co/colors and keeping a record of which ones were _already_ being used. for now this is good tho! thanks! 🎉

**Comment by sohilpandya** [comment]
I can certainly look into this and pick a set of colours :)

**Review by iteles — APPROVED** [review]
Agreed with @nelsonic but great! Thanks @sohilpandya

**Pull request merged**